php/base/Exchange.php

```diff
@@ -1158,7 +1158,7 @@ public function __construct($options = array()) {
 
         $this->precisionMode = DECIMAL_PLACES;
         $this->paddingMode = NO_PADDING;
-        $this->number = \Closure::fromCallable('floatval');
+        $this->number ='floatval';
```

Comment:
If we do this, then it becomes untranspileable: 

- https://github.com/ccxt/ccxt/blob/master/js/kraken.js#L487
- https://github.com/ccxt/ccxt/blob/master/js/phemex.js#L730
- https://github.com/ccxt/ccxt/blob/master/js/bl3p.js#L91

Reply:
check latest updates